packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/color-picker.styles.ts

```diff
@@ -0,0 +1,138 @@
+import { css } from "@microsoft/fast-element";
+
+export const ColorPickerStyles = css`
+
+    .root .colorUI
+    {
+        display:none;
+        padding: 2px;
+        flex-direction:row;
+        background-color: var(--neutral-layer-floating);
+        border: calc(var(--outline-width) * 1px) solid var(--neutral-outline-rest);
+        border-radius: calc(var(--corner-radius) * 1px);
+    }
+
+    .root.open .colorUI
+    {
+        display:flex;
+        position:absolute;
+        z-index:1;
+        margin-left:-32px;
+    }
+
+    .pickerContainer
+    {
+        margin: 4px 6px 4px 4px;
+    }
+
+    .inputContainer
+    {
+        width: 65px;
+        margin: 0 4px 4px 0;
+    }
+
+    .satLightPicker
+    {
+        position:relative;
+        width: 200px;
+        height: 200px;
+        margin-bottom: 17px;
+        background: -moz-linear-gradient(bottom, #000 0%, rgba(0, 0, 0, 0) 100%), -moz-linear-gradient(left, #FFF 0%, rgba(255, 255, 255, 0) 100%);
+        background: -webkit-linear-gradient(bottom, #000 0%, rgba(0, 0, 0, 0) 100%), -webkit-linear-gradient(left, #FFF 0%, rgba(255, 255, 255, 0) 100%);
+        background: -ms-linear-gradient(bottom, #000 0%, rgba(0, 0, 0, 0) 100%), -ms-linear-gradient(left, #FFF 0%, rgba(255, 255, 255, 0) 100%);
+        background: -o-linear-gradient(bottom, #000 0%, rgba(0, 0, 0, 0) 100%), -o-linear-gradient(left, #FFF 0%, rgba(255, 255, 255, 0) 100%);        
```

Comment:
Can these all just referenced `linear-gradient`? If we're using custom elements then all browsers in our support matrix should support the non-prefixed version.

Reply:
```suggestion
        background: linear-gradient(bottom, #000 0%, rgba(0, 0, 0, 0) 100%), -moz-linear-gradient(left, #FFF 0%, rgba(255, 255, 255, 0) 100%);
```

Reply:
I had struggled with this at first. As it seemed that Edge wasn't working without the -webkit- prefix. However, I found that it was the directions ("bottom","left") needed the "to" keyword to work correctly. This allowed me to remove all of the prefix versions. So, done!